From: Michael Tokarev <mjt@tls.msk.ru>
Subject: disable macro-diagnistics feature from uuid
Date: Fri, 29 Nov 2024 13:02:51 +0300

This feature is disabled in rust-uuid in debian,
so do not use it.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -51,3 +51,3 @@
     "fast-rng",
-    "macro-diagnostics",
+#    "macro-diagnostics",
 ]
